scripts/activate.sh

```diff
@@ -38,8 +38,8 @@ read -p "NPM 2FA Code: " twofactorcode
 for env in $envs; do
     echo npm dist-tag add $module@$version "$tag-$env" --otp="$twofactorcode";
     npm dist-tag add $module@$version "$tag-$env" --otp="$twofactorcode";
+    $DIR/grabthar-verify-npm-publish "$version" "$tag-$env";
```

Comment:
My only concern with doing this here is, the 2FA code may expire if we have too much time in this loop. Can we publish the dist tags, then loop again and run `grabthar-verify-npm-publish`?